tests/test_scheduler.py

```diff
@@ -203,6 +203,33 @@ def test_lock_acquisition_with_auto_start(self):
             self.assertEqual(mocked.call_count, 1)
             self.assertEqual(stopped_process.is_alive.call_count, 1)
 
+    def test_lock_release(self):
+        """Test lock acquisition"""
```

Comment:
Mind updating this to `Test that scheduler.release_locks() only releases acquired locks`?